From a346aaf148dc2ce53da30e2f67223834495c0fd6 Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Mon, 22 Oct 2012 13:26:46 -0400
Subject: [PATCH 126/278] g726dec: do not validate sample rate

For decoding it does not really matter what the sample rate is.
---
 libavcodec/g726.c |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/libavcodec/g726.c b/libavcodec/g726.c
index ed4bd26..91b5e54 100644
--- a/libavcodec/g726.c
+++ b/libavcodec/g726.c
@@ -422,14 +422,6 @@ static av_cold int g726_decode_init(AVCodecContext *avctx)
 {
     G726Context* c = avctx->priv_data;
 
-    if (avctx->strict_std_compliance >= FF_COMPLIANCE_STRICT &&
-        avctx->sample_rate != 8000) {
-        av_log(avctx, AV_LOG_ERROR, "Only 8kHz sample rate is allowed when "
-               "the compliance level is strict. Reduce the compliance level "
-               "if you wish to decode the stream anyway.\n");
-        return AVERROR(EINVAL);
-    }
-
     avctx->channels       = 1;
     avctx->channel_layout = AV_CH_LAYOUT_MONO;
 
-- 
1.7.9.5

